corehq/apps/export/views/list.py

```diff
@@ -1012,7 +1013,7 @@ class ODataFeedListView(BaseExportListView, ODataFeedListHelper):
 
     @property
     def lead_text(self):
-        return _("""
+        return _(format_html("""
```

Comment:
I think `_` and `format_html` need to be swapped here? The translation system looks for `ugettext("...")` (or its alias, here `_`) to identify the translation to begin with, and the string to translate is the template string not the one with the variables plugged, right? If I'm interpreting right, `format_html(_("""..."""), ...)`should work just fine

Reply:
Yep, thanks!